### ISSUE TITLE
rosbag_rewrite_types_rules.json should include all HLP (rosjava) message topics that are in use

### ISSUE THREAD

**Issue by trey0** [nasa/astrobee#521]
Jonathan found a bag from Kibo-RPC Astrobee ISS guest science activity that contained the `/command` topic, which was a bad topic (published by rosjava on the HLP) but not in the rewrite rules.

As explained in [Using Astrobee Robot Telemetry](https://nasa.github.io/astrobee/html/using_telemetry.html), bad topics that aren't rewritten can cause Python scripts processing the bag to crash with the error message

> genmsg.msg_loader.MsgNotFound: Cannot locate message [Header]: unknown package [std_msgs] on search path [{}]

which matches what he saw. And you can figure out what the bad topics are with

> rosrun bag_processing rosbag_detect_bad_topics.py in.bag

Here are my recommended steps to address this issue:

- [x] Add the `/command` topic to the rewrite rules.
- [x] Audit the HLP code to check if there are any other missing bad topics. I originally guessed that all the topics produced by the baseline FSW on the HLP might match the wildcard rule `/gs/*`, which worked for all the ISS bags I tested before, but evidently there may be some exceptions. Does anybody have insight into this?
- [x] Check if we can publicly release this bag so we can include it in the bag_processing regression tests in the open source repo. (It's not critical if we can't, just a good best practice. We have a script that strips down an example bag to only one message on each topic so the bags don't waste too much repo space.)

**Comment by trey0** [comment]
I realized that the third task listed above is not actually useful because the regression test bags are generated by the `rosbag_sample.py` script, which has to fix the message definitions in the process of sampling messages, so any test run on these bags to check whether we fix all the message definitions is doomed to "succeed".

Marking that task closed.

**Comment by trey0** [comment]
Task two was auditing the code to find more message topics published on the HLP. I found extra topics and added some of them in #564.

During this process, it became obvious that there are a few message topics that can be published both from the HLP and from the MLP, like `/mgt/cpu_monitor/state`, and "fixing" the message definition for old messages published on the MLP can break them if there has been a schema change since the message was recorded. Luckily, the unit testing caught that during the `rosbag_verify.py` deserialization step. I dealt with it by adding a check that should hopefully avoid accidentally fixing MLP messages.

I also found more HLP messages published by the [isaac_gs_ros_bridge](https://github.com/nasa/isaac/tree/master/apks/isaac_gs_ros_bridge), but I'm not worrying about adding those because we don't currently have any plans to use that bridge again, and it seems like its messages weren't recorded during previous ISS activities.

Marking task two done.
